tests/components/fitbit/test_sensor.py

```diff
@@ -217,7 +217,7 @@ async def test_device_battery_level(
     assert state.attributes == {
         "attribution": "Data provided by Fitbit.com",
         "friendly_name": "Charge 2 Battery",
-        "icon": "mdi:battery-50",
+        "icon": "mdi:battery-60",
```

Comment:
Why does this change?

Reply:
Is it due to using battery level key value for the icon while we used to use the battery key value, which is used for the state?

Reply:
It's weird:, the state is a string right now "medium".

I'd like to make this match the current design standards to use a battery device class. I was hesitant to just break it given our new deprecation rules, but happy to fix this either now or in future PRs.

(I can also revert this since it's not just a refactor.)

Reply:
Yeah, I suggest waiting with changing the icon until we change the state of the sensor to use the battery level instead of the string.